packages/multiple-choice/src/multiple-choice.jsx

```diff
@@ -111,7 +111,7 @@ export class MultipleChoice extends React.Component {
         feedback,
         value: choice.value,
         correctness:
-          checked && isEvaluateMode ? correctness(choice.correct) : undefined,
+          (checked || choice.correct ) && isEvaluateMode ? correctness(choice.correct && checked) : undefined,
```

Comment:
this is starting to lose readability.. how about: 
```suggestion
          isEvaluateMode ? correctness(choice.correct, checked) : undefined
        
```

```javascript
const correctness = (isCorrect, isChecked) => isChecked ? (isCorrect ? 'correct' : 'incorrect' ): undefined;
```